policy/datasets.yaml

```diff
@@ -564,6 +564,25 @@ src:
     python: lsst.afw.table.SourceCatalog
     tables: raw
     template: ''
+fakeSourceCat:
+    description: "A calexp sized catalog of fake sources"
```

Comment:
What does "calexp sized catalog" mean, is is somehow the size of a calexp? is it a calexp, sized catalog? Put in a more detailed description